accounting/receivables/customer_payments/online_payment.rst

```diff
@@ -0,0 +1,42 @@
+======================
+Invoice Online Payment
+======================
+
+To make it more convenient for your customers to pay the invoices you issue, you can activate the
+**Invoice Online Payment** feature, which adds a *Pay Now* button on their **Customer Portal**.
+
+.. image:: media/online-payment-acquirers.png
+   :align: center
+   :alt: Payment acquirer choice after having clicked on "Pay Now"
+
+Configuration
+=============
+
+Make sure your :ref:`Payment Acquirers are correctly configured <payment_acquirers/configuration>`.
+
+.. note::
+   By default, ":doc:`Wire Transfer <../../../general/payment_acquirers/wire_transfer>`" is the only
+   Payment Acquirer activated, but you still have to fill out the payment details.
+
+To activate the Invoice Online Payment, go to :menuselection:`Accounting --> Configuration -->
+Customer Payments`, enable **Invoice Online Payment**, and click on *Save*.
+
+Customer Portal
+===============
+
+After issuing the invoice, click on *Send & Print* and send the invoice by email to the customer.
+They will receive an email with a link to the see the invoice on their **Customer Portal**.
```

Comment:
```suggestion
They will receive an email with a link that redirects them to the invoice on their **Customer Portal**.
```
They will receive an email with a link to see the invoice on their **Customer Portal**.